### DISCUSSION TITLE
Script de arranque para los nodos

### DISCUSSION THREAD

**Discussion by rubensantibanezacosta** [opencanarias/taple-client#5] [Ideas]
Tal como me comentasteis, lo primero para trastear con Taple era arrancar los nodos. 

Pues, me preparé un pequeño Script, que monta previo cuestionario interactivo, una cantidad "n" de nodos utilizando un docker-compose.yml que se construye de forma dinámica. Os paso el [repositorio](https://github.com/rubensantibanezacosta/taple-local-automation).

Ahora toca entender como funciona la API, y si es que me estoy saltando pasos... Saludos

**Comment by FlorentinPG** [comment]
Gracias por tu aportación. Definitivamente es una buena idea, especialmente para tutoriales o casos muy controlados. La analizaremos para siguientes iteraciones.

Saludos.

**Comment by bcuestav** [comment]
Muchas gracias @rubensantibanezacosta . En la ULL estamos trabajando en una PoC con TAPLE y tu aportación nos ha ayudado a comenzar.
